docs/tutorials/install_and_admin/geonode_install/install_geonode_application.txt

```diff
@@ -90,14 +90,14 @@ GeoNode Setup
 
 Let's download GeoNode from the main GeoNode repository on GitHub:
 
-.. 
+..
     For the purpose of this training the GeoNode repository has
     been downloaded and placed in the `geonode` user `home folder` (/home/geo/geonode).
     *Skip* the following command group.
 
 ::
 
-    git clone https://github.com/GeoNode/geonode.git
+    git clone https://github.com/GeoNode/geonode.git --branch 2.6.x
```

Comment:
I see, I think those changes are good also for 2.4.x as well as master and eventually 2.6.x because the big change will be "migrate" instead of "syncdb" but is not in this PR. We can either merge this now with some minor changes or wait until 2.6 with an integration on "migrate"